npm/react-mount-utils/src/index.ts

```diff
@@ -1,12 +1,64 @@
 import * as React from 'react'
-import getDisplayName from './getDisplayName'
+import type { ReactNode } from 'react'
```

Comment:
All shared react utils are here now, all the react adapters are thin layers on top of this. This proved the easiest and cleanest way to solve this.